tests/monotouch-test/MediaToolbox/FormatNamesTest.cs

```diff
@@ -45,9 +52,14 @@ public void LocalizedNameForMediaType ()
 		[Culture ("en")]
 		public void LocalizedNameForMediaSubType ()
 		{
+#if MONOMAC
+			if (!TestRuntime.CheckMacSystemVersion (10, 11))
```

Comment:
Same: replace `CheckSystemVersion` with call to `TestRuntime.CheckXcodeVersion`.